packages/linked-deb/linked-deb.pacscript

```diff
@@ -0,0 +1,9 @@
+name="linked-deb"
+gives="linked"
+version="1.4.13"
+replace="${gives}"
+url="https://github.com/lostdesign/linked/releases/download/v${version}/linked_${version}_amd64.deb"
+description="An easy, distraction-free way to record your thoughts, declutter your mind and keep the things you want to remember. We all only have so much room in our brains for things. Let Linked be the place to put something you want to remember and let you free up space for deeper thinking."
```

Comment:
Can you shorten this any more?

Reply:
I copied it from the official website. If I take the short one, that's too short. 
**"linked - daily journaling without distraction"**
Is this okay?